models/blobs/ethereum/enriched.sql

```diff
@@ -0,0 +1,45 @@
+{{ config(
+    schema = 'blobs_ethereum',
+    alias = 'blobs_enriched',    
```

Comment:
We may rename the file and model name as:
`models/blobs/ethereum/blobs_ethereum_blobs_enriched.sql`
`blobs_ethereum_blobs_enriched`

Reply:
Ah yes please go ahead, I currently still get some unique key errors, but should be resolved soon